README.md

```diff
@@ -72,120 +73,121 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Corrected dutch to german. Thanks [@damienleroy](https://github.com/damienleroy)!
+- Added the Portuguese language accents. Thanks [@pcanavar](https://github.com/pcanavar)!
+- Fixed positioning of toolbar on scaled desktops.
 
 ### Screen Ruler
 
-- Improved UI/UX and settings descriptions.
-- Fixed utility exit logic to close all threads and avoid crashes. (This was a hotfix for 0.62)
+- Improved the acrylic brush used in the menu. Thanks [@niels9001](https://github.com/niels9001)!
 
 ### Settings
 
-- UI icons updated. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
-- Descriptions improvement and disambiguation. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
-- Fixed checkbox margins and other design tweaks. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+- Added a feature to backup/restore settings to/from a file. Thanks [@jefflord](https://github.com/jefflord)!
+- Fixed an issue causing shortcuts shown in OOBE not updating to new values when the window was re-opened.
+- Fixed the "Documents" folder usage in the backup/restore feature. Thanks [@davidegiacometti](https://github.com/davidegiacometti!
 
 ### Text Extractor
 
-- Removed extra spaces when recognizing Chinese, Japanese or Korean languages. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
-- Fixed utility exit logic to close all threads and avoid crashes. (This was a hotfix for 0.62)
-- Fixed an issue where a selection would start on right-click.
+- Added a warning about how to install languages for OCR recognition.
+- Fixed the overlay not focusing after the first activation.
+- Added spaces between CJK and non-CKJ words. Thanks [@maggch97](https://github.com/maggch97)!
 
-### Installer
+### Video Conference Mute
+- Added a setting to hide the Video Conference Mute overlay when muted. Thanks [@akabhirav](https://github.com/akabhirav)!
+- Added a warning about deprecating Video Conference Mute in the future (v0.67), please check https://github.com/microsoft/PowerToys/issues/21473 for more information.
 
-- Added logic to exit PowerToys on upgrade before trying to update .NET.
-- Updated the .NET dependency to 6.0.9.
-- Added clearer installation step names for the bootstrapper. Thanks [@htcfreek](https://github.com/htcfreek) and [@Jay-o-Way](https://github.com/Jay-o-Way)!
-- Windows App SDK, VC++ redistributable and PowerToys Interop runtime files are now shared between utilities through hardlinks, reducing installation size.
-
-### Documentation
+### Installer
 
-- Fixed typos in Keyboard Manager documentation. Thanks [@eltociear](https://github.com/eltociear)!
-- Replaced docs.microsoft.com links with learn.microsoft.com. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Added some missing items that were causing Settings and PowerRename issues on some configurations.
```

Comment:
Hm, this is just a bit vague...

Reply:
Installer was missing some dependencies that made the applications not work as expected. :)

Reply:
Rephrased it a bit.